engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetInputTextureFromFbo.java

```diff
@@ -108,7 +110,7 @@ public SetInputTextureFromFbo(int textureSlot, SimpleUri fboUri, FboTexturesType
 
         this.material = getMaterial(materialUrn);
 
-        update(); // Cheeky way to initialise textureId
+        propertyChange(null); // Cheeky way to initialise textureId
         fboManager.subscribe(this);
```

Comment:
Shouldn't we subscribe to the POST_FBO_REGENERATION event here too?

Reply:
This is weird.
In my workspace, I already subscribe to POST_FBO_REGENERATION.

Reply:
Strange. It seems ok now as I review your last commit. Never mind then.

Reply:
Oooh. This is weird again. If I look at the diff of the individual commit it seems ok. But if I see all the changes at once, the problem is still there. Weird. Let me do some additional checks..

Reply:
Ok, I think we are getting confused because there are two nearly identical constructors, which also points at why duplicated code is bad. Specifically, by the look of it, the second constructor should be calling the first instead of duplicating all the work.

Reply:
Oh boy, I feel ashamed.

Either way, it should be fixed now :)